src/components/Cards/PlayedCards/PlayedCards.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styles from '../Cards.module.css';
+
+export class PlayedCards extends React.Component {
+    render() {
+        return (
+            <ul id="played-cards">
+                {this.props.cards.map(card => {
+                    return <img id={"card-" + card.id} className={styles.allCards}  alt={"card-" + card.id} key={card.id} src={require(`../cards/card (${card.id}).jpg`)}/>
```

Comment:
You could have this as a "one-liner", you don't need the return here.

Reply:
A Pure Component or? @Ro4052

Reply:
I meant just the map. I wouldn't bother making this a Pure component, if you want to optimise, you'd just make this a function component.